build.gradle

```diff
@@ -14,13 +14,11 @@ repositories {
 }
 
 dependencies {
-    testImplementation 'junit:junit:4.12'
-
-    implementation 'com.squareup.okhttp3:okhttp:4.9.0'
-    implementation 'org.apache.httpcomponents:httpclient:4.5.9'
     implementation 'com.googlecode.json-simple:json-simple:1.1.1'
-    implementation 'com.squareup.okhttp3:mockwebserver:4.9.0'
```

Comment:
Only seems to be used in unit tests, so the testImplementation configuration should suffice.

However, when I removed the implementation configuration, Java complained that org.jetbrains:annotations @TestOnly was missing. It seems likely that org.jetbrains:annotations was a transitive dependency of com.squareup.okhttp3:mockwebserver in the implementation configuration.